pkgs/development/python-modules/numba/default.nix

```diff
@@ -70,18 +79,35 @@ in buildPythonPackage rec {
     done
   '';
 
-  # Copy test script into $out and run the test suite.
+  # run a smoke test in a temporary directory so that
+  # a) Python picks up the installed library in $out instead of the build files
+  # b) we have somewhere to put $HOME so some caching tests work
+  # c) it doesn't take 6 CPU hours for the full suite
   checkPhase = ''
```

Comment:
Custom `checkPhase`s should begin with `runHook preCheck` and end with `runHook postCheck`, even if those hooks don't exist (yet).